### PR TITLE
Two new Target files for Action1 and Level RMM agents

### PR DESCRIPTION
**Pull request by dfirtnt**
## Description

Please include a summary of the change and (if applicable) which issue is fixed.

## Checklist:
Please replace every instance of `[ ]` with `[X]` OR click on the checkboxes after you submit your PR

- [X] I have generated a unique `GUID` for my Target(s)/Module(s)
- [X] I have placed the Target(s)/Module(s) in an appropriate subfolder in Targets or Modules. If one doesn't exist, I have either added it to the `Misc` folder or created a relevant subfolder **with justification**
- [X] I have set or updated the version of my Target(s)/Module(s)
- [X] I have verified that KAPE parses the Target(s)/Module(s) successfully via kape.exe, using `--tlist`/`--mlist` and corrected any errors 
- [X] I have validated my Target(s)/Module(s) against test data and verified they are working as intended
- [X] I have made an attempt to document the artifacts within the Target(s) or Module(s) I am submitting. If documentation doesn't exist, I have placed `N/A` underneath the Documentation header
- [X] For Targets, I have consulted either the [Target Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/TargetGuide.guide), [Target Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/TargetTemplate.template), [Compound Target Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/CompoundTargetGuide.guide), or [Compound Target Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/CompoundTargetTemplate.template) to ensure my Target(s) follow the same format
- [ N/A ] For Modules, I have consulted either the [Module Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/ModuleGuide.guide), [Module Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/ModuleTemplate.template), [Compound Module Guide](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/CompoundModuleGuide.guide), or [Compound Module Template](https://github.com/EricZimmerman/KapeFiles/blob/master/Modules/CompoundModuleTemplate.template) to ensure my Module(s) follow the same format

If your submission involves an SQLite database, have you considered making an [SQLECmd Map](https://github.com/EricZimmerman/SQLECmd/tree/master/SQLMap/Maps) for the SQLite database? If you make a Map, please add the SQLite database to the [SQLiteDatabases.tkape Compound Target](https://github.com/EricZimmerman/KapeFiles/blob/master/Targets/Compound/SQLiteDatabases.tkape).

Thank you for your submission and for contributing to the DFIR community!

### CODE REVIEW

File: Action1.tkape [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RecreateDirectories: true
+Targets:
+    -
+        Name: Action1 Client Application logs
+        Category: ApplicationLogs
+        Path: C:\Windows\Action1\logs
+        FileMask: '*.log'
+        Comment: "Contains Application Log entries such as service start and incomming connections, and deployed scripts/jobs."
+
+
```
Comment (AndrewRathbun) [inline Action1.tkape:14]: Will need to remove this line

File: Level.tkape [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Id: 5e2c322f-616c-42e4-9cd7-4546cf2412e6
+RecreateDirectories: true
+Targets:
+    -
+        Name: Action1 RMM Client Application logs
+        Category: ApplicationLogs
+        Path: C:\Program Files\Level
+        FileMask: '*.log'
+        Comment: "Contains Application Log entries such as service start and incomming connections."
+
```
Comment (AndrewRathbun) [inline Level.tkape:13]: Need to remove this line

File: Level.tkape [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+# Documentation
+# https://www.crowdstrike.com/blog/analysis-of-intrusion-campaign-targeting-telecom-and-bpo-companies/
```
Comment (AndrewRathbun) [inline Level.tkape:16]: Need a new line at the end of the file

File: Action1.tkape [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Documentation
+# https://dfirtnt.wordpress.com/2023/08/23/rmm-action1-client-side-evidence/
+# https://www.bleepingcomputer.com/news/security/hackers-start-abusing-action1-rmm-in-ransomware-attacks/
```
Comment (AndrewRathbun) [inline Action1.tkape:17]: Need a new line at the end of the file

File: Action1.tkape [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Description: Action1 Application Logs
```
Comment (AndrewRathbun) [inline Action1.tkape:1]: This needs to go into the Targets folder and into an appropriate subfolder

File: Level.tkape [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Description: Level.io Application Logs
```
Comment (AndrewRathbun) [inline Level.tkape:1]: This needs to go into the Targets folder and into an appropriate subfolder

**Review by AndrewRathbun — CHANGES_REQUESTED** [review]
Please see comments

File: Action1.tkape (hunk 5 above)
Comment (dfirtnt) [reply Action1.tkape:1]: I'm new to Git. I will resolve these and send a new request with the corrections. Thanks!!

Comment (AndrewRathbun) [reply Action1.tkape:1]: Sounds good. Let me know if you have any questions. Best of luck!

**Review by AndrewRathbun — APPROVED** [review]
Much better, thank you! Great work

**Comment by AndrewRathbun** [comment]
@dfirtnt one last thing, can you make sure KAPE doesn't throw errors with `--tlist`? That's one of the checks on the Pull Request checklist. I can't check until the morning.

**Comment by dfirtnt** [comment]
@AndrewRathbun   I was able to successfully run kape.exe --tlist .\Apps. with no errors. Thanks!

**Pull request merged**